src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationemployment/api/integration/resource/SARTestData.kt

```diff
@@ -14,6 +14,9 @@ object SARTestData {
   val profileRequestOfKnownPRN = makeProfileRequestDTO(TestData.createProfileJsonRequest)
   val profileJsonOfKnownPRN = objectMapper.valueToTree<JsonNode>(profileRequestOfKnownPRN).get("profileData")
 
+  val anotherPRN = "K9876BC"
```

Comment:
Should this be a constant?

Reply:
I want a secondary known profile for another set of testing as the data patterns are different (more history items)

Reply:
I just switch all "history" test cases to this "another" profile